prov/efa/src/rxr/rxr_pkt_cmd.c

```diff
@@ -541,6 +552,12 @@ void rxr_pkt_handle_send_completion(struct rxr_ep *ep, struct fi_cq_data_entry *
 	case RXR_DC_EAGER_TAGRTM_PKT:
 		/* no action to be taken here */
 		break;
+	case RXR_DC_MEDIUM_MSGRTM_PKT:
+		/* no action to be taken here */
```

Comment:
Don't we need to call a function here to increase ```tx_entry->bytes_acked```?

Also ```rxr_pkt_handle_ctrl_sent``` need to be updated as well.